### ISSUE TITLE
Cannot change device if connection fails

### ISSUE THREAD

**Issue by dominikargast** [Esri/Trek2There#80]
If connection to an external device fails (e.g. if the device does not accept any more connections), an error message is shown and the "internal" radio button is selected. Dismissing the error message and selecting the "external" radio button leads to a new connection attempt, which fails again. In this situation, it is impossible to choose another device, see screenshot below.

<img width="483" alt="screen shot 2018-05-08 at 10 43 56" src="https://user-images.githubusercontent.com/25575734/39731783-113df6a8-52ad-11e8-9d12-8f25aa93fafd.png">

**Comment by marikavertzonis** [comment]
testing with app build 3.0.23
 - seems this unable to connect message only appears on **android**. i see it on the trimble tdc, sony, pixel 2 and when i dismiss it i can now select another receiver (which is the point of this issue). 
 - i haven't been able to replicate this behavior on **ios**. on ios you only see he devices that are successfully paired and connected (which weeds out the chance of attempting to connect to a device that cant be connected too, before you even try)
 - on **windows** the app 'locks up' and has the message of 'not responding'. theres no dialog to dismiss and you cant just choose another device. the only thing to do is close the app.
 - on **uwp**, after saying no to all the pairing requests, the app crashes

in summary - good for android and ios, still an issue for windows and uwp.

**Comment by dominikargast** [comment]
@marikavertzonis please create a new issue for the windows/uwp case and close this one if verified. The windows/uwp issues are crashes that are not related to this issue.
